tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -2121,6 +2126,26 @@ int CorProfiler::GetProbesStatuses(WCHAR** probeIds, int probeIdsLength, debugge
     return 0;
 }
 
+//
+// Fault-Tolerant Instrumentation methods
+//
+void CorProfiler::ReportSuccessfulInstrumentation(ModuleID moduleId, int methodToken, const WCHAR* instrumentationVersion,
+    int products)
+{
+    const auto& instrumentationVersionString = shared::WSTRING(instrumentationVersion);
```

Comment:
nit: the `&` is not needed here since you create a new object.

Reply:
Done in 9a92692d26bede161b4c38dab24df7c7395bc3bb.